### PR TITLE
Add move_to_element to sel.text & Update CheckboxTable locator on host drift page

### PR DESCRIPTION
**Pull request by jkrocil**
{{pytest: -v -k "host_drift"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d31f8131f69cca4401de910455f19dc176b674f7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/fixtures/pytest_selenium.py [hunk 1]
```diff
@@ -524,7 +524,7 @@ def text(loc, **kwargs):
 
     Returns: A string containing the text of the element.
     """
-    return element(loc, **kwargs).text
+    return move_to_element(loc, **kwargs).text
```
Comment (psav) [inline cfme/fixtures/pytest_selenium.py]: @seandst @mfalesni  just a double check here. Do you foresee any issues with this?

Comment (seandst) [inline cfme/fixtures/pytest_selenium.py]: :+1:

Comment (mfalesni) [inline cfme/fixtures/pytest_selenium.py]: I see just slight slowdown here. Nothing special otherwise.

**Pull request merged**